src/components/views/rooms/RoomTile.js

```diff
@@ -179,15 +184,18 @@ module.exports = React.createClass({
         var notificationCount = this.props.room.getUnreadNotificationCount();
```

Comment:
why are some notifications considered by the parent (via this.props.highlight) and some right here? It's rather confusing.